src/platform/datapath_winkernel.c

```diff
@@ -2087,8 +2187,6 @@ QuicDataPathSocketReceive(
                     Binding->Datapath->DatagramStride);
         }
 
-        continue;
-
```

Comment:
Why did you remove this? If we're pending, shouldn't we be doing this?

Reply:
No, we still need to hit one of the checks in the drop case to release the buffers we copied.